### ISSUE TITLE
Combine breaking

### ISSUE THREAD

**Issue by Darkrune-Wakeflame** [KitaenSilva/OSFOP-Project-Repo#55]
Using the table in the starting room to attempt combining the branch and alcohol puts you into a menu that has neither item. Exiting this menu then breaks all of the tables in the house so you can't combine the items and progress. 
![blaugh](https://cloud.githubusercontent.com/assets/26861264/25168924/d4393f3c-24aa-11e7-81c0-d4a9b1d5202b.PNG)

However using literally any other table will AUTO combine both items and give you the bottle and the wet branch. 

One minor glitch is that when you combine items this way, then quickly exit to title and reload a save, it will pull up the graphics of both items being received, but luckily does not give duplicates of the items.

**Comment by KitaenSilva** [comment]
Yeah, stuff like this is why I relegated combining to only that one table - I had no idea how it would cooperate, if at all, so I needed it to be _optional_ for now, until the kinks are ironed out.
